Server/src/main/java/local/AssetDAOImpl.java

```diff
@@ -190,6 +192,31 @@ public String getAssetTypeNameFromID(String assetTypeID) {
         return assets;
     }
 
+
+    public ArrayList<Asset> getAllLiveAssets(String sort) {
+        ArrayList<Asset> assets = new ArrayList<>();
+        ResultSet rs;
+        switch (sort){
+            case "Descending RUL":
```

Comment:
don't hard code string values

Reply:
I am not hardcoding string values I am checking if the sort type selected is Descending than I use a query, if not then I use the original query